api/v1alpha1/flowcollector_types.go

```diff
@@ -227,6 +227,10 @@ type FlowCollectorConsolePlugin struct {
 	// HPA spec of an horizontal pod autoscaler to set up for the plugin Deployment.
 	// +optional
 	HPA *FlowCollectorHPA `json:"hpa,omitempty"`
+
+	// Additional port name to use in the console
+	// +optional
+	PortsName map[string]string `json:"portsname,omitempty"`
```

Comment:
`PortsName` or `PortNames` ? I think the latter is more correct(?)
By the way, what's the way to disable port naming? Providing an empty object, as opposed to an absent field (which would rather show default names) ?

Reply:
Yes, I will change this to PortNames.

My understanding of the goal of this story is to add the possibility to add custom name for ports, a custom name defined here would have priority over default IANA naming.

Do you think we should also provide the possibility to disable the port naming feature?

Reply:
Ok, I missed that part. I will add an option.